### ISSUE TITLE
when the text editor reload a new project, prettier won't format the code files

### ISSUE THREAD

**Issue by zourdycodes** [prettier/prettier#11337]
So when I tried to figure it out whether there might be my code editor is caused the error or something like that but after 5 times attempt I gave up. the message on the navigation bottom bar is (cannot format the file.xyz, error).

I use visual studio code-insiders newest version, anyway I have contacted the vs code support but they said it's probably the extension.

**Comment by sosukesuzuki** [comment]
I don't know why it happened, but if it's caused by an extension, you should report an issue to https://github.com/prettier/prettier-vscode

**Comment by zourdycodes** [comment]
yep, I think its caused by the extension because I have been re-install the vs-code both the insiders and the visual studio but nothing changed it's always wouldn't format my code on the other hand if I reset my code-formatted on vs-code settings it worked just fine but as  I said every time I start a new project that issues always showed up and it's a bit annoying. 

I am trying to figure out (on a prettier codebase) how this happens when I was in my free time.
